engine/engine.go

```diff
@@ -31,10 +31,12 @@ const (
 
 // Config are global configuration parameter for logserver
 type Config struct {
-	ContentBatchSize int           `json:"content_batch_size"`
-	ContentBatchTime time.Duration `json:"content_batch_time"`
-	SearchMaxSize    int           `json:"search_max_size"`
-	CacheExpiration  time.Duration `json:"cache_expiration"`
+	ContentBatchSize  int           `json:"content_batch_size"`
+	ContentBatchTime  time.Duration `json:"content_batch_time"`
+	SearchMaxSize     int           `json:"search_max_size"`
+	CacheExpiration   time.Duration `json:"cache_expiration"`
+	ExcludeExtensions []string      `json:"exclude_extensions"`
```

Comment:
Maybe easier to use a "glob" style excluding. then there is no need to differentiate between directories and extensions.

Reply:
Yes there is. 
For example, "bin" as an extension should be filtered, but bin as a dir might not.

Reply:
For extensions use: `*.bin`
For directories use: `*/dirname/*`

Reply:
See example of glob usage here: https://github.com/Stratoscale/logserver/blob/9b25e37bab162578b1ab5a80214aeee757adba73/parse/journal.go#L17